package.json

```diff
@@ -52,7 +52,7 @@
     "pre-commit": "^0.0.9",
     "tape": "^3.0.3",
     "tape-cluster": "2.1.0",
-    "tchannel": "^2.5.1",
+    "tchannel": "2.5.1",
```

Comment:
This is a dev dependency of tchannel. A recent tchannel update broke tests. This is in an effort to pass this test run and potentially fix in another PR. You can ignore this change for now.